Packs/CommonDashboards/ReleaseNotes/1_2_2.md

```diff
@@ -0,0 +1,4 @@
+
+#### Dashboards
+##### SLA
+Added new widgets to the 'SLA' dashboard.
```

Comment:
I would say something like The dashboard now support new triage and containment SLA's

Reply:
Done